integration

```diff
@@ -94,6 +94,7 @@
   "B5r1oJ0A9G/teufel_raumfeld",
   "bacco007/sensor.opennem",
   "bacco007/sensor.waternsw",
+  "baldarn/whatsapper",
```

Comment:
This is not a integration, for containers, look at the add-on documentation https://developers.home-assistant.io/docs/add-ons

Reply:
thanks for your time.
so this is not mergable? what can I do in order to have this in the HACS repo?

the intent is to have something like `manish-custom-notifier` but using a local whatsapp proxy

Reply:
i'm i missing something?

Reply:
That is not what you have requested to add with this PR.

Reply:
i see. what do i have to change? the name?

Reply:
in the link i linked the integration repo